src/charm.py

```diff
@@ -910,8 +910,11 @@ def _refresh_firewall(self):
                 FirewallEntry.decode(entry.strip())
                 for entry in firewall_denylist_config.split(",")
             ]
+        allowlist = [
+            FirewallEntry.decode(str(entry.host)) for entry in self._state.ssh_debug_infos
+        ]
         firewall = Firewall("lxdbr0")
-        firewall.refresh_firewall(denylist)
+        firewall.refresh_firewall(denylist, allowlist)
```

Comment:
```suggestion
        firewall.refresh_firewall(denylist=denylist, allowlist=allowlist)
```
You might want to use kwargs as guards here to avoid using the wrong position ;-)

Reply:
fixed, thank you!